### PR TITLE
Add Poolside FM 1.0

### PR DESCRIPTION
**Pull request by mfpierre**

### CODE REVIEW

File: Casks/poolside-fm.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'poolside-fm' do
+  version '1.0'
+  sha256 'dc508e85636d6a616a4c8ef504f4c0327e81e5b3a40ba8f44be22d987d2e482a'
+
+  url 'https://s3-eu-west-1.amazonaws.com/poolside-fm/HeyThereHotStuff.zip'
```
Comment (vitorgalvao) [inline Casks/poolside-fm.rb]: Needs a `  # amazonaws.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/7c010b42bd7ac679c1f002a0368b36bb3f495612/doc/CASK_LANGUAGE_REFERENCE.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

File: Casks/poolside-fm.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'poolside-fm' do
+  version '1.0'
```
Comment (vitorgalvao) [inline Casks/poolside-fm.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

**Comment by mfpierre** [comment]
@vitorgalvao Thanks for the feedback, I've made the modifications

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by mfpierre** [comment]
@vitorgalvao :white_check_mark:

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**